### ISSUE TITLE
Add smarter statement linter

### ISSUE THREAD

**Issue by no-reply** [rightsstatements/data-model#23]
Statements and their URIs have a number of clear "MUST" in the technical paper and some other implied requirements. We should be able to verify at least the following:
- Presence of:
  - a version number
  - exactly one `skos:definition` (per language)
  - at least one `skos:scopeNote`
- Trailing slash on URIs
- Matching `dct:identifier` with the one reflected in the statement URI
- Membership in a concept scheme with the same version
- Membership in a collection (is this required?)

Feel free to expand on this list in comments.

**Comment by aisaac** [comment]
Thanks for starting it!
The one I can think of right now is the presence of exactly one |skos:prefLabel| (per language)
There might be more, but I won't have time for this sort of investigation before ten days now, I'm afraid :-(

**Comment by anarchivist** [comment]
> Membership in a collection (is this required?)

I think it's implicit, but I'd say yes (cc @musebrarian). 

Other requirements:
- All members of a concept scheme must be represented within the Turtle
- All members of a collection must be represented within the Turtle
- If a language tag is used on any literal, it must be used consistently across all the statements (in other words, "partial translations" are not acceptable)

**Comment by anarchivist** [comment]
(I'd like @mzeinstra's and @aisaac's  feedback on the last point in particular.)

**Comment by aisaac** [comment]
I agree with the need of avoiding partial translations, but it may be not trivial to check, for the case where we can have several statements of a same property applied to a right statement (would we count them and check the counts across languages?)

Antoine

**Comment by no-reply** [comment]
> (would we count them and check the counts across languages?)

I sense this is the best we could do. Taking `@en` as the expected number, and failing or warning if there is a different count of notes.

Alternatively, we could consider a translation "complete" if it has a `prefLabel` and a `description`.  It might be useful to get feedback from the other group on this---how do we plan to manage translations?

**Comment by aisaac** [comment]
Could we just postpone it until the creation of translation becomes a more pressing issue, or should it be addressed right now?

**Comment by anarchivist** [comment]
I think it could be postponed, or at least separated from the more generalized need for a linter. I think there was some discussion about ensuring that the rights statement application would only serve up "complete" translations - so there needs to be some way for us to make that determination - or to guard that during the editorial process.

**Comment by no-reply** [comment]
:+1: to postponing. It's a fairly minor issue, and it shouldn't be hard to come upon an answer once someone is actually working on translations.
